packages/clerk-js/src/core/resources/SignIn.ts

```diff
@@ -184,6 +191,23 @@ export class SignIn extends BaseResource implements SignInResource {
     }
   };
 
+  public startSamlFlow = async (params: StartSamlFlowParams): Promise<void> => {
+    const { redirectUrl, redirectUrlComplete } = params || {};
+
+    const { firstFactorVerification } = await this.prepareFirstFactor({
```

Comment:
⛔ This is essentially the only difference with `authenticateWithRedirect` method. What's a differentiator we can use to call prepareFirstFactor instead of create and merge these two methods into one?

Reply:
For parity with the SU case we could add a `continueSignIn` prop.

Reply:
Added `continueSignIn` for now, will need to revisit.

Reply:
Removed it